src/components/ProjectMembersSummary/ProjectMembersSummary.tsx

```diff
@@ -20,6 +20,7 @@ export const ProjectMembersSummary: FC<ProjectMembersSummaryProps> = ({
 
   return (
     <MemberListSummary
+      total={members?.total}
```

Comment:
Changed this because using length might not be accurate could be more than the default query amount and it could get paginated to be shorter. I can put the couple files effected by this in a separate pr if it's easier to look at.

Reply:
Yeah that's a good catch. It's fine in this PR. I would prefer in a separate commit; commits should be changing a single logical thing.

Reply:
I cleaned up the commit history so this is in a separate commit